### DISCUSSION TITLE
Where to post about documentation errors

### DISCUSSION THREAD

**Discussion by raiAmagi** [dagster-io/dagster#25272] [Q&A]
Where to post about documentation errors

We are currently learning by reading the documentation and running sample PGs as well.

If I notice what I believe to be an error in the description, where should I post my question?

I posted it in the FAQ before, but is this the right place?

I am using a translation site to write the sentences.
Sorry if the text is rude.

**Comment by garethbrickman** [comment]
You can report any feedback or issues with the documentation using this template: https://github.com/dagster-io/dagster/issues/new?assignees=&labels=area%3A+docs&projects=&template=report_documentation_issue.yml

**Reply by raiAmagi** [reply]
Thanks for letting me know where to report.
I will report from here.
